app/models/bookable_slot.rb

```diff
@@ -14,6 +14,16 @@ def pm?
     PM.pm?(self.end)
   end
 
+  def appointments
+    scope = Appointment.where(location_id: schedule.location_id)
+
+    if am?
```

Comment:
Looking at this now I reckon we should collapse it down to:

```ruby
ending_at = am? end_at - 1.minute : end_at

scope.where(proceeded_at: start_at..ending_at)
```

What say you?